src/Components/Footer.js

```diff
@@ -0,0 +1,153 @@
+import React from "react";
+import styled from "styled-components";
+
+const Footer = () => {
+  return (
+    <FooterContainer>
+      <FooterWrap>
+        <LeftCaption>
+          <div className="topCaption">
```

Comment:
스타일드 컴포넌트로 만들어 보면 어떨까요
```suggestion
          <topCaption>
```